Toolbox/arcsdm/areafrequency.py

```diff
@@ -40,14 +37,15 @@
 VAT.next() returns (ID, VALUE, COUNT)
 
 """
+
 # Import modules
 
 class UserException(Exception):
     pass
 
-
 import sys, string, os, random, traceback, tempfile
 import arcgisscripting
```

Comment:
I don't know whether a geoprocessing context / geoprocessor (for lack of better wording) created with arcgisscripting.create() needs to be created by the developer or not. Esri documentation makes no reference to it, so probably not? Digging through some of the arcpy source code, I see internal calls to it, so it might be something that is handled by their code internally and shouldn't be messed with by us. I wish Esri was clearer about it.

The blog post about arcgisscripting does mention the geoprocessor, but in the example, it is imported from arcpy.geoprocessing (on which there is no documentation by Esri, to my knowledge). My head hurts trying to understand what the correct way to use this stuff is. 😅